infrastructure/src/main/java/org/corfudb/infrastructure/configuration/ServerConfiguration.java

```diff
@@ -0,0 +1,487 @@
+package org.corfudb.infrastructure.configuration;
+
+
+import ch.qos.logback.classic.Level;
+import io.netty.channel.EventLoopGroup;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.corfudb.comm.ChannelImplementation;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This class holds various server configuration parameters.
+ *
+ * <p>Created by maithem on 12/4/19.
+ */
+
+public class ServerConfiguration extends PropertiesConfiguration {
+    // Server general parameters
+    private static final String SERVER_DIR = "serverDirectory";
```

Comment:
This is a great class. nit: If we add some relevant comments to these constants it would be even better. It then starts serving as another starting point to read and understand code.

Reply:
There will be an .ini file that will be loaded into this class. That config file will be documented.